src/openzaak/components/zaken/tests/test_notifications_send.py

```diff
@@ -898,33 +898,6 @@ def _reset_debug():
             os.environ["DEBUG"] = "no"
             self.addCleanup(_reset_debug)
 
-    def test_invalid_notification_config_create(self, mock_notif):
```

Comment:
The test tries to set an api_root of `bla/`, but `ape_pie.APIClient` doesn't complain about this upon instantiation, only when you make the request (because then `requests.get` will complain about it):

`*** requests.exceptions.MissingSchema: Invalid URL 'bla/foo': No scheme supplied. Perhaps you meant https://bla/foo?`

Reply:
Ok, when I think the commented test can be removed